libcontainerd/supervisor/remote_daemon.go

```diff
@@ -17,9 +17,9 @@ import (
 	"github.com/docker/docker/pkg/pidfile"
 	"github.com/docker/docker/pkg/process"
 	"github.com/docker/docker/pkg/system"
+	"github.com/moby/buildkit/util/grpcerrors"
```

Comment:
Also /cc @dmcgowan who's looking at unifying some error-handling code and types (just in case any of this relates to that)